pkgs/development/interpreters/bats/libraries.nix

```diff
@@ -25,12 +25,12 @@
 
   bats-file = stdenv.mkDerivation rec {
     pname = "bats-file";
-    version = "0.3.0";
+    version = "0.4.0";
```

Comment:
Follow the guidelines for commits.

`bats.libraries.bats-file: 0.3.0 -> 0.4.0`

Reply:
Done, ty!